documentation/docs/discovery.mdx

```diff
@@ -1,127 +1,106 @@
 
-Open Payments defines the following additional claims:
+In many cases a user will have multiple accounts and will register a single
+short and easy to transcribe **Personal Payment Pointer** which they link to
+their default account. This Personal Payment Pointer can then be easily
+exchanged with counter-parties when initiating a payment (either as sender or
+receiver).
 
-- `issuer`  
-  URL of the server's sessions endpoint where the client is able to create new
-  sessions with the `subject`.
-  
-- `authorization_issuer`  
-    URL of the server's sessions endpoint where the client is able to create new
-    sessions with the `subject`.
+> Example Personal Payment Pointer:  
+> `$wallet.example/alice`
 
-- `authorization_endpoint`  
-    URL of the server's sessions endpoint where the client is able to create new
-    sessions with the `subject`.
+Personal Payment Pointers are analogous to an email address or payment card
+number. Email addresses are given to counter-parties so that they can begin
+sending emails to the target inbox. Similarly, a payment card number can be
+given to a counter-party so that they can request payment from the target
+account.
 
-- `token_endpoint`  
-    URL of the server's sessions endpoint where the client is able to create new
-    sessions with the `subject`.
+Users can share their Personal Payment Pointers to both get paid and make
+payments using Open Payments.
 
-- `invoices_endpoint`  
-  URL of the server's invoices endpoint where the client is able to create new
-  invoices with the `subject`.
+Importantly, a Payment Pointer is not sensitive in the same way that a payment
+card number is. It can be shared freely with no risk that funds will be pulled
+from the target account without the account owner's consent
 
-- `mandates_endpoint`  
-  URL of the server's mandates endpoint where the client is able to create new
-  mandates with the `subject`.
-
-- `assets_supported`
-  A list of asset definitions for assets that can be used to create agreements
-  on this server. The schema of an asset definition is defined in
-  [Assets](#assets).
-
-Other claims MAY also be returned.
+## Metadata
 
-Claims that return multiple values are represented as JSON arrays. Claims with
-zero elements MUST be omitted from the response.
+Account metadata is provided by an Open Payments Server as a JSON document that
+describes the various service endpoints that are available to:
 
-An error response uses the applicable HTTP status code value.
+- connect to the account via Interledger
+- get authorization to perform actions on the account (using OAuth 2.0)
+- get more information about the owner of the account (using OpenID Connect)
+- create invoices and mandates that are linked to the account
 
-The following is a non-normative example response:
+The document is hosted at the account URL and is the default representation of
+the account resource (using the media type of `application/json`).
```

Comment:
Can we specify this type to be `application/open-payments`? In the Java Connector something like `/accounts/{accountId}` is the Admin API endpoint for that account, and already responds to `application/json` with the actual account-settings.

If we want to re-use that endpoint for public consumption, I think we need a different content-type.

Reply:
> In the Java Connector something like /accounts/{accountId} is the Admin API endpoint for that account

That's a good point. Payment pointer domain + `/accounts/...` may already be in use. Alternatively, what about putting the API behind `/open-payments/accounts` or some more specific well known URL? (recalling back to Ian's feedback about using paths vs differentiating based on content types)

Reply:
+1 on this. On first reading I was concerned about the same thing.

Reply:
> If we want to re-use that endpoint for public consumption, I think we need a different content-type.

Why would you re-use an existing endpoint for a new service? Just use account URLs that are different to the admin API URLs?

> Payment pointer domain + /accounts/... may already be in use

The API can be behind whatever URL you want. It's up to the wallet to decide that and then construct Payment Pointers that reflect that.

There is no requirement that the account URL be `payment pointer domain + /accounts/`

Reply:
@adrianhopebailie I'm not opposed to what you're saying, I'm just unclear around how to make it work. If my payment pointer is `david$xpring.money`, then where should I host my OpenPayments API paths?

Reply:
See https://github.com/interledger/open-payments/issues/40